2019-12-15 08:36:44,170 INFO: Microblog startup [in C:\Users\lzhan\Documents\GitHub\nnhsse-flask\luke-home\microblog\app\__init__.py:45]
2019-12-15 08:37:01,527 INFO: Microblog startup [in C:\Users\lzhan\Documents\GitHub\nnhsse-flask\luke-home\microblog\app\__init__.py:45]
2019-12-15 08:38:21,614 INFO: Microblog startup [in C:\Users\lzhan\Documents\GitHub\nnhsse-flask\luke-home\microblog\app\__init__.py:45]
2019-12-15 08:39:11,660 INFO: Microblog startup [in C:\Users\lzhan\Documents\GitHub\nnhsse-flask\luke-home\microblog\app\__init__.py:45]
2019-12-15 08:39:36,938 INFO: Microblog startup [in C:\Users\lzhan\Documents\GitHub\nnhsse-flask\luke-home\microblog\app\__init__.py:45]
2019-12-15 08:39:58,863 INFO: Microblog startup [in C:\Users\lzhan\Documents\GitHub\nnhsse-flask\luke-home\microblog\app\__init__.py:45]
2019-12-15 08:44:10,074 INFO: Microblog startup [in C:\Users\lzhan\Documents\GitHub\nnhsse-flask\luke-home\microblog\app\__init__.py:45]
2019-12-15 08:47:04,836 INFO: Microblog startup [in C:\Users\lzhan\Documents\GitHub\nnhsse-flask\luke-home\microblog\app\__init__.py:45]
2019-12-15 09:03:34,084 INFO: Microblog startup [in C:\Users\lzhan\Documents\GitHub\nnhsse-flask\luke-home\microblog\app\__init__.py:45]
2019-12-15 09:06:10,502 INFO: Microblog startup [in C:\Users\lzhan\Documents\GitHub\nnhsse-flask\luke-home\microblog\app\__init__.py:45]
2019-12-15 09:08:39,420 INFO: Microblog startup [in C:\Users\lzhan\Documents\GitHub\nnhsse-flask\luke-home\microblog\app\__init__.py:45]
2019-12-15 09:10:00,152 INFO: Microblog startup [in C:\Users\lzhan\Documents\GitHub\nnhsse-flask\luke-home\microblog\app\__init__.py:45]
2019-12-15 09:11:38,819 INFO: Microblog startup [in C:\Users\lzhan\Documents\GitHub\nnhsse-flask\luke-home\microblog\app\__init__.py:45]
2019-12-15 09:15:13,111 INFO: Microblog startup [in C:\Users\lzhan\Documents\GitHub\nnhsse-flask\luke-home\microblog\app\__init__.py:45]
2019-12-15 09:21:19,270 INFO: Microblog startup [in C:\Users\lzhan\Documents\GitHub\nnhsse-flask\luke-home\microblog\app\__init__.py:45]
2019-12-15 09:22:30,086 INFO: Microblog startup [in C:\Users\lzhan\Documents\GitHub\nnhsse-flask\luke-home\microblog\app\__init__.py:45]
2019-12-15 09:27:58,690 INFO: Microblog startup [in C:\Users\lzhan\Documents\GitHub\nnhsse-flask\luke-home\microblog\app\__init__.py:45]
2019-12-15 09:28:04,047 ERROR: Exception on /lzhang3/followers [GET] [in c:\users\lzhan\documents\github\nnhsse-flask\luke-home\microblog\venv\lib\site-packages\flask\app.py:1891]
Traceback (most recent call last):
  File "c:\users\lzhan\documents\github\nnhsse-flask\luke-home\microblog\venv\lib\site-packages\flask\app.py", line 2446, in wsgi_app
    response = self.full_dispatch_request()
  File "c:\users\lzhan\documents\github\nnhsse-flask\luke-home\microblog\venv\lib\site-packages\flask\app.py", line 1951, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "c:\users\lzhan\documents\github\nnhsse-flask\luke-home\microblog\venv\lib\site-packages\flask\app.py", line 1820, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "c:\users\lzhan\documents\github\nnhsse-flask\luke-home\microblog\venv\lib\site-packages\flask\_compat.py", line 39, in reraise
    raise value
  File "c:\users\lzhan\documents\github\nnhsse-flask\luke-home\microblog\venv\lib\site-packages\flask\app.py", line 1949, in full_dispatch_request
    rv = self.dispatch_request()
  File "c:\users\lzhan\documents\github\nnhsse-flask\luke-home\microblog\venv\lib\site-packages\flask\app.py", line 1935, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "c:\users\lzhan\documents\github\nnhsse-flask\luke-home\microblog\venv\lib\site-packages\flask_login\utils.py", line 261, in decorated_view
    return func(*args, **kwargs)
  File "C:\Users\lzhan\Documents\GitHub\nnhsse-flask\luke-home\microblog\app\routes.py", line 161, in followers
    followers=followers)
  File "c:\users\lzhan\documents\github\nnhsse-flask\luke-home\microblog\venv\lib\site-packages\flask\templating.py", line 140, in render_template
    ctx.app,
  File "c:\users\lzhan\documents\github\nnhsse-flask\luke-home\microblog\venv\lib\site-packages\flask\templating.py", line 120, in _render
    rv = template.render(context)
  File "c:\users\lzhan\documents\github\nnhsse-flask\luke-home\microblog\venv\lib\site-packages\jinja2\asyncsupport.py", line 76, in render
    return original_render(self, *args, **kwargs)
  File "c:\users\lzhan\documents\github\nnhsse-flask\luke-home\microblog\venv\lib\site-packages\jinja2\environment.py", line 1008, in render
    return self.environment.handle_exception(exc_info, True)
  File "c:\users\lzhan\documents\github\nnhsse-flask\luke-home\microblog\venv\lib\site-packages\jinja2\environment.py", line 780, in handle_exception
    reraise(exc_type, exc_value, tb)
  File "c:\users\lzhan\documents\github\nnhsse-flask\luke-home\microblog\venv\lib\site-packages\jinja2\_compat.py", line 37, in reraise
    raise value.with_traceback(tb)
  File "C:\Users\lzhan\Documents\GitHub\nnhsse-flask\luke-home\microblog\app\templates\_follower.html", line 19, in template
    {% else if not user.is_following(follower) %}
  File "c:\users\lzhan\documents\github\nnhsse-flask\luke-home\microblog\venv\lib\site-packages\jinja2\environment.py", line 1005, in render
    return concat(self.root_render_func(self.new_context(vars)))
  File "C:\Users\lzhan\Documents\GitHub\nnhsse-flask\luke-home\microblog\app\templates\followers.html", line 14, in root
  File "C:\Users\lzhan\Documents\GitHub\nnhsse-flask\luke-home\microblog\app\templates\base.html", line 14, in root
    <span class="icon-bar"></span>
  File "c:\users\lzhan\documents\github\nnhsse-flask\luke-home\microblog\venv\lib\site-packages\flask_bootstrap\templates\bootstrap\base.html", line 10, in root
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
  File "c:\users\lzhan\documents\github\nnhsse-flask\luke-home\microblog\venv\lib\site-packages\flask_bootstrap\templates\bootstrap\base.html", line 20, in block_doc
    {% block body -%}
  File "c:\users\lzhan\documents\github\nnhsse-flask\luke-home\microblog\venv\lib\site-packages\flask_bootstrap\templates\bootstrap\base.html", line 39, in block_html
  File "c:\users\lzhan\documents\github\nnhsse-flask\luke-home\microblog\venv\lib\site-packages\flask_bootstrap\templates\bootstrap\base.html", line 92, in block_body
  File "C:\Users\lzhan\Documents\GitHub\nnhsse-flask\luke-home\microblog\app\templates\base.html", line 81, in block_content
  File "C:\Users\lzhan\Documents\GitHub\nnhsse-flask\luke-home\microblog\app\templates\followers.html", line 26, in block_app_content
  File "c:\users\lzhan\documents\github\nnhsse-flask\luke-home\microblog\venv\lib\site-packages\jinja2\environment.py", line 830, in get_template
    return self._load_template(name, self.make_globals(globals))
  File "c:\users\lzhan\documents\github\nnhsse-flask\luke-home\microblog\venv\lib\site-packages\jinja2\environment.py", line 804, in _load_template
    template = self.loader.load(self, name, globals)
  File "c:\users\lzhan\documents\github\nnhsse-flask\luke-home\microblog\venv\lib\site-packages\jinja2\loaders.py", line 125, in load
    code = environment.compile(source, name, filename)
  File "c:\users\lzhan\documents\github\nnhsse-flask\luke-home\microblog\venv\lib\site-packages\jinja2\environment.py", line 591, in compile
    self.handle_exception(exc_info, source_hint=source_hint)
  File "c:\users\lzhan\documents\github\nnhsse-flask\luke-home\microblog\venv\lib\site-packages\jinja2\environment.py", line 780, in handle_exception
    reraise(exc_type, exc_value, tb)
  File "c:\users\lzhan\documents\github\nnhsse-flask\luke-home\microblog\venv\lib\site-packages\jinja2\_compat.py", line 37, in reraise
    raise value.with_traceback(tb)
  File "C:\Users\lzhan\Documents\GitHub\nnhsse-flask\luke-home\microblog\app\templates\_follower.html", line 19, in template
    {% else if not user.is_following(follower) %}
  File "c:\users\lzhan\documents\github\nnhsse-flask\luke-home\microblog\venv\lib\site-packages\jinja2\environment.py", line 497, in _parse
    return Parser(self, source, name, encode_filename(filename)).parse()
  File "c:\users\lzhan\documents\github\nnhsse-flask\luke-home\microblog\venv\lib\site-packages\jinja2\parser.py", line 901, in parse
    result = nodes.Template(self.subparse(), lineno=1)
  File "c:\users\lzhan\documents\github\nnhsse-flask\luke-home\microblog\venv\lib\site-packages\jinja2\parser.py", line 883, in subparse
    rv = self.parse_statement()
  File "c:\users\lzhan\documents\github\nnhsse-flask\luke-home\microblog\venv\lib\site-packages\jinja2\parser.py", line 130, in parse_statement
    return getattr(self, 'parse_' + self.stream.current.value)()
  File "c:\users\lzhan\documents\github\nnhsse-flask\luke-home\microblog\venv\lib\site-packages\jinja2\parser.py", line 223, in parse_if
    drop_needle=True)
  File "c:\users\lzhan\documents\github\nnhsse-flask\luke-home\microblog\venv\lib\site-packages\jinja2\parser.py", line 164, in parse_statements
    self.stream.expect('block_end')
  File "c:\users\lzhan\documents\github\nnhsse-flask\luke-home\microblog\venv\lib\site-packages\jinja2\lexer.py", line 384, in expect
    self.name, self.filename)
jinja2.exceptions.TemplateSyntaxError: expected token 'end of statement block', got 'if'
2019-12-15 09:30:49,412 INFO: Microblog startup [in C:\Users\lzhan\Documents\GitHub\nnhsse-flask\luke-home\microblog\app\__init__.py:45]
